docs/toc.yml

```diff
@@ -122,7 +122,7 @@
         displayName: event-based activation
       - name: Run code on document open
         href: develop/run-code-on-document-open.md
-    - name: Manifest
+    - name: Manifest and app package
       items:
       - name: Overview
```

Comment:
If you accept the suggestion to reorder this section, consider renaming this to "Office Add-ins manifest" or "Manifest overview" since it doesn't mention app package.